src/core/stream_recv.c

```diff
@@ -873,14 +873,10 @@ QuicStreamRecvFlush(
     while (FlushRecv) {
```

Comment:
If we go with my other proposed changes, I think `FlushRecv` is now too ambiguous, and should be renamed to `IndiciateRecvEvents` to indicate that we can continue to indicate up receives to the app, but not necessarily that we can process receive completes yet.